src/status_im/ui/screens/multiaccounts/sheets.cljs

```diff
@@ -1,6 +1,7 @@
 (ns status-im.ui.screens.multiaccounts.sheets
   (:require [quo.core :as quo]
             [re-frame.core :as re-frame]
+            [status-im2.events]
```

Comment:
could you elaborate pls?

Reply:
This namespace is using the old events namespace, I added this dependency to ensure when I moved an event to status-im2 it was correctly loaded.

But now this is not relevant since I reverted those changes 3dc2cea849b74a2f8ba5bfa08c87e09f2d17c7f9